src/box/lua/net_box.lua

```diff
@@ -749,6 +775,8 @@ end
 
 function remote_methods:ping(opts)
     check_remote_arg(self, 'ping')
+    check_param_table(opts, REQUEST_OPTION_TYPES)
+    check_opts_buffer(opts)
```

Comment:
You also need to add the check to `space_begin`, `space_commit`, and `space_rollback`. It'd also be nice to check `connect()` options.